scripts/system_performance/tests/tests_system_performance_queries.py

```diff
@@ -0,0 +1,206 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+
+"""
+Test cases for system_performance_queries script
+"""
+
+import unittest
+from mock import patch, Mock, MagicMock
+
+from scripts.system_performance.system_performance_queries import DatabaseMonitorChecks
+from utils.settings import MYSQL_SETTINGS
+from utils.clients.connection_exception import ConnectionException
+
+
+class MockConnection(Mock):
+    """Mock object class"""
+    pass
+
+
+class TESTDatabaseMonitorChecks(unittest.TestCase):
+    """Test class for DatabaseMonitorChecks class"""
+
+    def setUp(self):
+        """Setup of DatabaseMonitor class call and default arguments dictionary"""
+        self.db_monitor_checks = DatabaseMonitorChecks()
+        self.arguments_dict = {'selection': 'run_number',
+                               'status_id': 4,
+                               'retry_run': '',
+                               'anomic_aphasia': 'finished',
+                               'end_date': 'CURDATE()',
+                               'interval': 1,
+                               'time_scale': 'DAY',
+                               'start_date': None,
+                               'instrument_id': None}
+
+    @patch('utils.clients.database_client.DatabaseClient.connect', return_value=MockConnection())
+    # pylint: disable=no-value-for-parameter
+    # pylint: disable=R0201
```

Comment:
combine these into one line: `# pylint: disable=no-value-for-parameter,no-self-use